migrations/20190107123190-create-profile.js

```diff
@@ -28,6 +28,9 @@ module.exports = {
         as: 'userId'
       }
     },
+    category: {
+      type: Sequelize.STRING
```

Comment:
Category should probably be ENUM, since `STEAM` is desired. Unless you are looking into adding extra categories asides the ones already provided.

Reply:
ok. noted will appropriate adjustment now